pkg/controller/pod_test.go

```diff
@@ -119,7 +119,9 @@ var _ = Describe("Dynamic Attachment controller", func() {
 						fakecri.NewFakeRuntime(*pod),
 						fakemultusclient.NewFakeClient(
 							networkConfig(multuscni.CmdAdd, "net1", macAddr),
-							networkConfig(multuscni.CmdDel, "net0", "")),
+							networkConfig(multuscni.CmdDel, "net0", ""),
+							networkConfig(multuscni.CmdAdd, "net0", macAddr),
```

Comment:
why do you need to mock adding net0 ? 

IIUC your test removed net0 and adds net1 ?... Both of which are already mocked.

Reply:
That's correct sorry, I was probably trying something and I forgot to remove this part.